harness/determined/exec/harness.py

```diff
@@ -36,6 +36,7 @@
 import simplejson
 
 import determined as det
+import determined_common
 from determined import gpu, horovod, layers, load, workload
 from determined_common import constants, storage
```

Comment:
non blocking

just a style thing (and for consistency w/ the CLI change), but should this instead be:
`from determined_common import constants, set_logger, storage`
and then do:
`set_logger(debug)`

(or vice versa, i.e., change the CLI to match the pattern here)

Reply:
In general, we try to follow the google style guide, which calls for importing modules and calling functions from their modules.

I'm a little lax about it when I'm in a file where we're really inconsistent.  But you're right, I should use `determined_common.set_logger` in both places.